### PR TITLE
Clarify readme and cleanup notebooks

### PR DESCRIPTION
**Pull request by IAlibay**
Towards 1.1.0 release

* Add extra details to README to detail the contents/layout of the workshop
* Add maint tool to clear outputs of practical notebooks (not applied to lectures as outputs can be useful there).

### CODE REVIEW

**Comment by IAlibay** [comment]
Sorry folks, it's late, I'm sure it's full of typos :(

File: README.md [hunk 1]
```diff
@@ -2,12 +2,123 @@
 
 ## Contents
 
-This repository contains the materials for the MDAnalysis 2021 PRACE/SURF workshop.
+This repository contains the materials for the 3 day MDAnalysis 2021 PRACE/SURF
+workshop.
```
Comment (orbeckst) [inline README.md]:
```suggestion
This repository contains the materials for the 3-day [MDAnalysis 2021 PRACE/SURF
workshop](https://www.mdanalysis.org/2021/04/09/prace-workshop/).
```

**Review by orbeckst — APPROVED** [review]
Thanks for doing all the work while we still know what happened. 

(Once this is merged I would raise an issue to make all the notebooks and PDFs links into the repo... but I don't want to throw this at you, anyone who feels like it could do that.)

**Comment by IAlibay** [comment]
> Thanks for doing all the work while we still know what happened.
> 
> (Once this is merged I would raise an issue to make all the notebooks and PDFs links into the repo... but I don't want to throw this at you, anyone who feels like it could do that.)

Thanks for reviewing @orbeckst, I've added links everywhere although there might be some style changes that folks might want to improve on. I'll merge for now, and let folks review/change before the next release.

**Pull request merged**